### ISSUE TITLE
Use unique jQuery file for the whole project

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#508]
```
I did a rough check on versions of jQuery we used for html5rocks project, and 
found that we are using different versions from different places in various 
files. Sometimes we fetched the jQuery file from 
http://ajax.googleapis.com/ajax/libs/jquery and sometimes we used local files 
under www.html5rocks.com, studio.html5rocks.com or playground.html5rocks.com.
I think it's better to use one unique file of jQuery.

Check 
here(http://www.google.com/codesearch?hl=en&sa=N&filter=0&q=jquery+package:http:
//html5rocks%5C.googlecode%5C.com) to see different versions of jQuery we uses.

```

Original issue reported on code.google.com by `han...@google.com` on 20 Apr 2011 at 6:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 20 Apr 2011 at 6:01

**Comment by GoogleCodeExporter** [comment]
```
This is fixed no?
```

Original comment by `ericbide...@html5rocks.com` on 28 Jun 2011 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed already.

-- 
Hanrui (�ߺ���)

Office Phone: 86 (10) 6250-3527
Cell Phone: +86 13810795320
```

Original comment by `han...@google.com` on 28 Jun 2011 at 2:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 8 Dec 2011 at 12:30
- Added labels: **Milestone-Q42011-1**
- Removed labels: **Milestone-6**

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 8 Dec 2011 at 7:27
- Added labels: **Milestone-Q12012**
- Removed labels: **Milestone-Q42011-1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 6 Mar 2012 at 7:03
- Changed state: **Fixed**
